### DISCUSSION TITLE
Regarding Security Vulnerabilities

### DISCUSSION THREAD

**Discussion by aashutoshrathi** [pallets/werkzeug#2616] [Q&A] [answered]
Hey! It seems we have few high security vulns for this package:
- https://security-tracker.debian.org/tracker/CVE-2023-25577
- https://security.snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319936

Wanted to know if there is a quick patch available for that.
Thanks

**Comment by davidism** [comment, marked as answer]
Look carefully at the reports and compare the fix version to the latest available version 😉

**Reply by aashutoshrathi** [reply]
Oh! missed it.
Thanks a ton!
